### ISSUE TITLE
(C++) Crash when parsing `->` with `indent_member_single` set to true

### ISSUE THREAD

**Issue by fletcher97** [uncrustify/uncrustify#3996]
Uncrustify crashes when using the arrow operator `->` along with `indent_member_single` set to true. Trying to generate logs doesn't work. The file is not created probably due to it crashing before it has a chance.

Tested with: `Uncrustify-0.76.0-128-1355077a` 

Steps to reproduce:
* Create a cpp file with the following content:
```cpp
#include <iostream>

class OtherClass
{
public:
	int i;
	OtherClass(): i(5) {}
	OtherClass* self() {return this;}
};

class MyClass
{
public:
	OtherClass x;
	int getI();
};

int MyClass::getI() {
	return this->x.self()->i;
}

int main() {
	MyClass c;
	std::cout << c.getI() << std::endl;
}
```
* create a config file with only the option `indent_member_single` set to `true`
* execute uncrustify:
  `uncrustify -c uncrustify.cfg -f test.cpp`

Expected output:
```
#include <iostream>

class OtherClass
{
public:
int i;
OtherClass() : i(5) {
}
OtherClass* self() {
	return this;
}
};

class MyClass
{
public:
OtherClass x;
int getI();
};

int MyClass::getI() {
	return this->x.self()->i;
}

int main() {
	MyClass c;
	std::cout << c.getI() << std::endl;
}
```
Actual output:
```
ParsingFrame::pop (indent_text:1351): orig line is   19, orig col is   30, type is       MEMBER,
ParsingFrame::pop (indent_text:1351): the type is MEMBER, is not coded. Please make a call.
```

Replacing the last arrow in `return this->x.self()->i;` by a dot `return this->x.self().i;` make it so uncrustify doesn't crash.

[//]: # "  If the issue is connected to existing Uncrustify options please, if possible, add the"
[//]: # "  following information to ease up the process:"
[//]: # "  • a link to a debug file:"
[//]: # "      generated with: 'uncrustify -p debug.txt -c pathToUsedConfig.cfg -f toBeFormatedFile.cpp' "
[//]: # "      Example: [debug.txt](https://linkToTheFile)"
[//]: # "      Example hosters for debug files: pastebin.com, gist.github.com, ..."
[//]: # "      The used config file is included in the debug file and does not need to be included here."
[//]: # 
[//]: # "  • include a small but complete test file that will be uncrustifyed"
[//]: # "  • include the generated results"
[//]: # "  • include the expected results"
[//]: # 
[//]: # "  ✋ please add a line containing ``` above and below of each of those three code sections"
[//]: # 
[//]: # "  • include the current version of your Uncrustify executable"
[//]: # "      printout via 'uncrustify -v'"
[//]: # "      Example: current version: uncrustify 0.63"
[//]: # "      or if possible additionally with the git sha of the commit"
[//]: # "                    current version: uncrustify 0.63 dc7b412"
[//]: # 
[//]: # "  • if possible include a version that worked"
[//]: # "      Example: working version: uncrustify 0.63"
[//]: # "      or"
[//]: # "                    working version: uncrustify 0.63 2a5e88f"

**Comment by micheleCTDEAdmin** [comment]
Fixed by PR#3997. 
@fletcher97 please check and close the issue if it works for you too.

**Comment by fletcher97** [comment]
Seems to work fine. Thanks for the quick fix
